Makefile

```diff
@@ -1,7 +1,26 @@
-#!/usr/bin/env bash
+##########################
+#  OpenShift Marketplace - Build and Test  #
+##########################
+
+SHELL := /bin/bash
+PKG   := github.com/operator-framework/operator-marketplace/pkg
```

Comment:
Probably something like `SRC_DIR`. Have a look at existing kube or openshift projects and use it as bolierplate, I think we should have checks for various environment related issues like 
- Is go installed?
- What version of go should I use?
- Is GOPATH defined?

Reply:
I agree, but I think it's outside the scope of this PR. I did just enough to generate the mocks I need. Making a full fledged Makefile warrants a new task in Jira and hence a new PR. Let me go and add this task in Jira.

Reply:
Then please add a TODO comment here and file a new Jira task